vendor/assets/javascripts/initializers.js

```diff
@@ -66,12 +66,21 @@ $(document).on('turbolinks:load', function() {
         $(clone).find('img').first().attr('src', res.attachment.icon.url);
         $(clone).first().removeClass('uk-hidden');
         UIkit.modal('#add-new-featured-image').hide();
+        $('.js-remove-image').removeClass('uk-hidden');
       });
   });
   $('.js-featured-image-table-element').each(function() {
     var imageUrl = $(this).data('imageUrl');
     $(this).find('input').on('click', function(event) {
       $('.js-featured-image').attr('src', imageUrl).removeClass('uk-hidden');
+      $('.js-remove-image').removeClass('uk-hidden');
     });
   });
+  $('.js-remove-image').each(function() {
+    $(this).find('a').on('click', function(event) {
+      $(this).find('input').first().val(1);
+      $('.' + this.dataset.target).addClass('uk-hidden');
```

Comment:
Instead of the `data-target` being a class name string, make it a full selector, so you don't have to add `.` and it can also work with IDs and attr selectors.